---
red_buoy_task:
  rotate_around_center:
    linear_velocity: 0.8
    angular_velocity: 0.8
  set_red_buoy_depth:
    sleep_duration: 3.0
  find_and_aim_red_buoy:
    full_rotation: false
    set_frame_duration: 4.0
    rotation_speed: 0.3
  wait_for_aligning_rotation_start:
    delay_time: 4.0
